aii-core/src/main/perl/Shellfe.pm

```diff
@@ -229,19 +231,23 @@ sub app_options
          HELP    => 'where to store lock files',
          DEFAULT => "/var/lock/quattor" },
 
+       { NAME    => 'parallel=i',
+         HELP    => 'run commands on hosts in parallel ',
+         DEFAULT => 0 },
+
+       { NAME    => 'logpid',
+         HELP    => "Add process ID to the log messages (disabled by default, unless parallel is > 1)",
+         DEFAULT => undef },
+
          # Options for osinstall plug-ins
        { NAME    => 'osinstalldir=s',
          HELP    => 'Directory where Kickstart files will be installed',
          DEFAULT => '/osinstall/ks' },
 
          # Options for DISCOVERY plug-ins
-         { NAME => 'dhcpconf=s',
-           HELP => 'name of dhcp configuration file',
-           DEFAULT => '/etc/dhcpd.conf' },
-
-         { NAME => 'restartcmd=s',
-           HELP => 'how to restart the dhcp daemon',
-           DEFAULT => 'service dhcpd restart' },
+       { NAME => 'dhcpcfg=s',
```

Comment:
In fact, this probably requires moving these options into `/etc/aii/aii-shellfe.conf` instead of `/etc/aii/aii-dhcp.conf`... This should be documented and `ncm-aiiserver` should be fixed to do it for sites that use it to manage their Quattor server (the recommended method IMO).